packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -138,7 +141,12 @@ const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
 
     const baseUrl = config.proxy.split(MOBIFY_PATH)[0]
     const privateClientEndpoint = `${baseUrl}${SLAS_PRIVATE_PROXY_PATH}`
-
+    if (refreshTokenTTL && refreshTokenTTL < 0) {
+        refreshTokenTTL = 0
+        throw new Error(
+            `'refreshTokenTTL' must be a non-negative value in seconds, overriding negative value ${refreshTokenTTL} to zero`
```

Comment:
We won't see the original `refreshTokenTTL` value in the error message since we override it before throwing the error

Reply:
Actually the logic here omits an important detail, you can't go **beyond** the default 90 day window from SLAS, so it needs to be `0` minimum and `90days` maximum

Reply:
good call, let's make that clear in the error message!

Reply:
Done ✅